services/ux-backend/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	handler "github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	"k8s.io/klog/v2"
+)
+
+var tokenLifetimeInHours int
+var listenPort int
+
+func init() {
+
+	var err error
+	tokenLifetimeInHours, err = strconv.Atoi(os.Getenv("ONBOARDING_TOKEN_LIFETIME"))
+	if err != nil || tokenLifetimeInHours <= 0 {
+		tokenLifetimeInHours = 48
+	}
+	klog.Info("generated token will be valid for ", tokenLifetimeInHours, " Hours")
+
+	defaultListeningPort := 8080
+	listenPortAsString := os.Getenv("UX_BACKEND_PORT")
+	if listenPortAsString == "" {
+		klog.Infof("No user-defined server listening port provided, defaulting to %d ", defaultListeningPort)
+		listenPort = defaultListeningPort
+	} else if listenPort, err = strconv.Atoi(listenPortAsString); err != nil {
+		klog.Errorf("Malformed user-defined listening port: %s. shutting down", listenPortAsString)
+		os.Exit(-1)
+	}
+
+	klog.Info("ux backend server listening on port ", listenPort)
+
+}
```

Comment:
This is not a good use case for init. Init has a finicky invocation time (the first time the package is referenced). Please move everything into `main`. I don't see a use case for `init` for this server.

Reply:
Done.